test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -196,29 +267,39 @@ contract("CoreIssuanceOrder", (accounts) => {
       });
     });
 
+    describe("when the order quantity is not a multiple of the natural unit of the set", async () => {
```

Comment:
In these specs where the setup is all common, can you help identify what is is you're toggling? I would put `generateFillOrderParameters`. In `beforeEach` on line 124, you're already doing this generation, and then you're regenerating for each of these test cases. I'd create a variable for each one and then just toggle the specific variable in each test case.